packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureGraph/to/helpers/V1_DomainBuilderHelper.ts

```diff
@@ -262,6 +264,23 @@ export const V1_buildAssociationProperty = (
     associatedProperty.type,
     `Association associated property 'type' field is missing`,
   );
+  const _class = context.graph.getMainGraphClass(associatedPropertyClassType);
+  const _association = context.graph.getOwnNullableAssociation(
```

Comment:
I dont think this is correct. Will this work if the association is part of a dependency?